_rules/text-contrast-afw4f7.md

```diff
@@ -52,6 +52,8 @@ For each test target, the [highest possible contrast](#highest-possible-contrast
 
 - [Success criterion 1.4.3: Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum) also has an exception for logos and brand names. Since logos and brand names are usually displayed through images to ensure correct rendering, this rule does not take logos or brand names into consideration. If a logo or brand name is included using [text nodes](https://dom.spec.whatwg.org/#text), this rule may produce incorrect results.
 
+- Text that has the same foreground and background color (a contrast ratio of 1:1) is not considered to be "visual presentation of text", making it inapplicable to the success criterion. Text hidden in this way can still cause accessibility issues under other success criteria, depending on the content.
```

Comment:
Should this be in "Accessibility Support" rather than in "Assumption"?

Reply:
Nope. Acc support is about techniques not working consistently between assistive tech. This is an interpretation of the success criterion. That belongs in assumptions.